test/adapters/prebid.spec.js

```diff
@@ -991,6 +999,26 @@ describe('ADAPTER: Prebid', function() {
             delete CONF.adapters[adapterID][CONSTANTS.CONFIG.SERVER_SIDE_ENABLED];
             done();
         });
+
```

Comment:
newly added unit test cases are disabled

Reply:
Yeah I am not sure how to access prebidConfig generated in fetchBids that's why disabled those test cases

Reply:
Enabled test cases